deploy/olm-catalog/hawtio-operator/0.2.0/hawtio-operator.v0.2.0.clusterserviceversion.yaml

```diff
@@ -242,6 +242,7 @@ spec:
           - console.openshift.io
           resources:
           - consolelinks
+          - consoleyamlsamples
```

Comment:
I'd suggest not to change the CSV for 0.2.0 version that's already been released. We can either add it later, or you can copy the CSV and create a new one for the upcoming release.

Reply:
Added v0.3.0

Reply:
Thanks. Could you remove the change from the 0.2.0 CSV?

Reply:
removed